### ISSUE TITLE
php73-pecl-apcu-bc

### ISSUE THREAD

**Issue by bdelamatre** [iusrepo/wishlist#235]
### What new package do you want?

php73-pecl-apcu_bc

### Why?

We need to support libraries that make use of apc_* functions. We are currently hosting applications that require apc_* functions using php 7.1, which was the last version of IUS to contain apcu_bc (https://github.com/iusrepo/wishlist/issues/114). We would like to upgrade those apps to PHP 7.3 as PHP 7.1 support ends this year.

### Testing

I agree to test the new package to ensure that it works as expected.  Once I am
satisfied with the results of my testing I will comment on this issue with the
word "STABLE" to get it promoted to the stable repos.

**Comment by carlwgeorge** [comment]
We've created a [php73-pecl-apcu-bc](https://github.com/iusrepo/php73-pecl-apcu-bc) package.  It is available from ius-testing on the new [IUS CDN](https://github.com/iusrepo/infrastructure/issues/14).  Please try it out and share your feedback.

**Comment by carlwgeorge** [comment]
@bdelamatre How is this package working for you?

**Comment by bdelamatre** [comment]
I apologize, this is still on my list to test. I think I need to adjust some scripts and policies before I can install and test it, since it looks like the CDN URLs changed.

**Comment by carlwgeorge** [comment]
Have you had a chance to test this yet?  Also, could I interest you in trying php72u-pecl-acpu-bc as well?  That was requested in #216, but the original requester no longer needs it.

**Comment by carlwgeorge** [comment]
@bdelamatre Still waiting on feedback for this package.  If you don't reply soon I'm going to remove the package from the testing repo and close this issue.

**Comment by carlwgeorge** [comment]
Due to lack of feedback, I have removed php73-pecl-apcu-bc from the testing repo.  The spec file is available [here](https://github.com/iusrepo/php73-pecl-apcu-bc), and the RPMs that were built are available [here](https://cirrus-ci.com/task/5741744604315648).  We can revive this in the future if there are people willing to test and provide feedback.

**Comment by pkrbj** [comment]
@carlwgeorge Please could the packages for apcu-bc be created for PHP73 and PHP74? I can't seem to download the artifacts in the link provided to try it out and test.

**Comment by carlwgeorge** [comment]
PHP 7.3 will be EOL in a [few months](https://www.php.net/supported-versions), so I'd prefer not to pursue php73-pecl-apcu-bc at this point.  If you are interested in php74-pecl-apcu-bc please open a separate issue.
